android/app/src/main/res/values-hu/strings.xml

```diff
@@ -619,15 +619,11 @@
 	<string name="bookmark_list_description_hint">Leírás hozzáadása (szöveg vagy html)</string>
 	<string name="not_shared">Privát</string>
 	<string name="speedcams_alert_title">Sebességmérő kamerák</string>
-	<!-- Generic «Always» string -->
-	<string name="always">Mindig</string>
-	<!-- Generic «Never» string -->
-	<string name="never">Soha</string>
 	<string name="place_description_title">A hely ismertetése</string>
 	<!-- this text will be shown in application notification preferences opposite checkbox which enable/disable downloader notifications. Devices on Android 8+ are affected. -->
 	<string name="notification_channel_downloader">Kártyák letöltése</string>
-	<!-- Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit -->
-	<string name="pref_tts_speedcams_auto">Figyelmeztetés ha van kockázat a sebesség korlátozás megsértéséről</string>
+	<!-- "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit -->
+	<string name="pref_tts_speedcams_auto">Figyelmeztetni a gyorshajtásra</string>
```

Comment:
```suggestion
	<string name="pref_tts_speedcams_auto">Gyorshajtási figyelmeztetés</string>
```

Reply:
> 🇭🇺 needs change

@d4f5409d: Changed to `Gyorshajtási figyelmeztetés`

![ios_speed_cam_prefs_hu1](https://github.com/organicmaps/organicmaps/assets/25888296/34f3cadb-30cb-49a4-9902-2b82a1dd1f2b)